github/table_github_search_code.go

```diff
@@ -117,9 +116,8 @@ func tableGitHubSearchCodeList(ctx context.Context, d *plugin.QueryData, h *plug
 func extractSearchCodeRepositoryFullName(_ context.Context, d *transform.TransformData) (interface{}, error) {
 	code := d.HydrateItem.(*github.CodeResult)
 	if code.HTMLURL != nil {
-		rx := regexp.MustCompile(`(?s)` + regexp.QuoteMeta("github.com/") + `(.*?)` + regexp.QuoteMeta("/blob"))
-		replacer := strings.NewReplacer("github.com/", "", "/blob", "")
-		return replacer.Replace(rx.FindString(*code.HTMLURL)), nil
+		rx := regexp.MustCompile(`https?://.+?/(.+?)/blob`)
```

Comment:
Have you tested this change (along with other `github_search_*` table changes) against a GitHub cloud instance for backward compatibility? I saw some query results in https://github.com/turbot/steampipe-plugin-github/pull/160#issuecomment-1119199458, but wasn't sure if those were for an enterprise or cloud instance.

Reply:
the tests were against github.com (`base_url = "https://api.github.com/`) and GH Enterprise with custom hostname (`base_url = "https://abc.xyz.gov/`. I guess our GHE is Server, not Cloud - I'm not clear on the difference.